addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2460,16 +2460,36 @@ const RangeUserValueWidget = UnitUserValueWidget.extend({
         let max = this.el.dataset.max && parseFloat(this.el.dataset.max) || 100;
         const step = this.el.dataset.step && parseFloat(this.el.dataset.step) || 1;
         this.displayValue = this.el.dataset.displayRangeValue;
+        this.appendValue = this.el.dataset.append || false;
+        this.prependValue = this.el.dataset.prepend || false;
+        this.exponential = false;
+        if (this.el.dataset.exponential) {
+            this.exponential = (value) => max ** ((value - max / 2) / (max / 2));
+        }
         if (min > max) {
             [min, max] = [max, min];
             this.input.classList.add('o_we_inverted_range');
         }
         this._setInputAttributes(min, max, step);
         this.containerEl.appendChild(this.input);
         if (this.displayValue) {
+            if (this.prependValue) {
+                this.prependEl = document.createElement("span");
+                this.prependEl.textContent = this.prependValue;
+                this.prependEl.classList.add("ms-2", "me-1");
+                this.containerEl.appendChild(this.prependEl);
+            }
             this.outputEl = document.createElement('output');
-            this.outputEl.classList.add('ms-2');
+            if (!this.prependValue) {
+                this.outputEl.classList.add("ms-2");
+            }
             this.containerEl.appendChild(this.outputEl);
+            if (this.appendValue) {
```

Comment:
That's not a unit, it's a coefficient. And when I use "data-unit", it fails, because it converts the value in unit with `weUtils.convertNumericToUnit`